lib/pronto/formatter/github_status_formatter/sentence.rb

```diff
@@ -0,0 +1,42 @@
+module Pronto
+  module Formatter
+    class GithubStatusFormatter
+      class Sentence
+        def initialize(words)
+          @words = words
+        end
+
+        def to_s
+          case words.size
+          when 0
+            ''
+          when 1
+            words[0].to_s.dup
+          when 2
+            "#{words[0]}#{WORD_CONNECTORS[:two_words_connector]}#{words[1]}"
+          else
+            to_oxford_comma_sentence
```

Comment:
I shamelessly ripped off and simplified the implementation from [ActiveSupport](http://apidock.com/rails/ActiveSupport/CoreExtensions/Array/Conversions/to_sentence).